ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -852,6 +852,30 @@ objects:
         EOF
         fi
 
+        if has_variant "rt-debug"; then
+          cat > ${manifests}/realtime-worker-machine-config.yml << EOF
+        apiVersion: machineconfiguration.openshift.io/v1
+        kind: MachineConfig
+        metadata:
+          labels:
+            machineconfiguration.openshift.io/role: worker
+          name: realtime-worker
+        spec:
+          kernelType: realtime
+        EOF
+          cat > ${manifests}/worker-kernel-debug-machine-config.yml << EOF
+        apiVersion: machineconfiguration.openshift.io/v1
+        kind: MachineConfig
+        metadata:
+          labels:
+            machineconfiguration.openshift.io/role: worker
+          name: kernel-debug
+        spec:
+          kernelArguments:
+          - 'debug'
```

Comment:
This looks likely to do something, I see
`debug		[KNL] Enable kernel debugging (events log level).`
but...are you sure the person isn't asking for `kernel-debug` which is an entirely separate kernel build that we're not including right now?  See https://fedoraproject.org/wiki/KernelDebugStrategy

Reply:
Oh I see
```
       debug
           Turn on debugging output. This is equivalent to systemd.log_level=debug. Note that this option is also read by the kernel itself and enables
           kernel debug output. Passing this option hence turns on the debug output from both the system manager and the kernel.
```